tests/unit/models/reddit/test_subreddit.py

```diff
@@ -123,3 +123,10 @@ def test__getitem(self):
         wikipage = subreddit.wiki["Foo"]
         assert isinstance(wikipage, WikiPage)
         assert "foo" == wikipage.name
+
+
+class TestSubredditModmailConversationsStream(UnitTest):
+    def test_random(self):
+        submodstream = self.reddit.subreddit("mod").mod.stream
+        submodstream.modmail_conversations()
+        assert submodstream.subreddit == Subreddit(self.reddit, "all")
```

Comment:
This could be simplified to `== "all"`.

In addition, I'd like a test that tests when the subreddit is `"Mod"` (with capitalization), to catch the behavior I described in an earlier comment.

Reply:
Will be fixed and new test added in my next commit.